testing/bdd/features/validation/sequence_id_num.feature

```diff
@@ -1,6 +1,11 @@
 @validation @syntax @sequence @xsd
 Feature: Sequence ID and Sequence Number
-  Both are mandatory parameters and the sequence number has to be a number (no letters)
+  Both are mandatory parameters and the sequence number has to be a number (no letters).
+  Moreover two documents with the same sequence identifier shall have different sequence numbers.
```

Comment:
Oh so indeed I missed that point, I can compare the produced xml of both document, and raise the error only if they are different.